mlflow/tracking/fluent.py

```diff
@@ -5,30 +5,50 @@
 
 from __future__ import print_function
 
-import atexit
 import numbers
 import os
+
+import atexit
 import sys
 import time
 
 from mlflow.entities import Experiment, Run, SourceType
+from mlflow.tracking.client import MlflowClient
 from mlflow.utils import env
 from mlflow.utils.databricks_utils import is_in_databricks_notebook, get_notebook_id, \
     get_notebook_path, get_webapp_url
 from mlflow.utils.mlflow_tags import MLFLOW_DATABRICKS_WEBAPP_URL, \
     MLFLOW_DATABRICKS_NOTEBOOK_PATH, \
     MLFLOW_DATABRICKS_NOTEBOOK_ID
 from mlflow.utils.validation import _validate_run_id
-from mlflow.tracking.client import MlflowClient
-
 
 _EXPERIMENT_ID_ENV_VAR = "MLFLOW_EXPERIMENT_ID"
 _RUN_ID_ENV_VAR = "MLFLOW_RUN_ID"
 _active_run = None
+_active_experiment_id = None
+
+
+def set_experiment(experiment_name):
```

Comment:
I wonder whether we should consider a version that takes experiment ID too. I guess it would be safest to call that something different to avoid confusion though, so we can do that later. I agree that set_experiment(name) will be more common.

Reply:
Do we want to call this method as `set_experiment_by_name` or `set_active_experiment_name`

Reply:
I think `set_experiment` and `set_experiment_by_id` might be reasonable?

Reply:
+1 to @aarondav 's suggestions. If we think set experiment using name will be more commonly  used, let's optimize for that.